cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -108,11 +119,14 @@ function FlyoutMenu( element, triggerSel, contentSel, altTriggerSel ) {
 
   /**
    * Expand animation has completed.
+   * Call deferred collapse function,
+   * if set (otherwise it will call a noop function).
    */
   function _expandEnd() {
     _isAnimating = false;
     _contentDom.removeEventListener( _transitionEndEvent, _expandEndBinded );
     this.dispatchEvent( 'expandEnd', { target: this } );
+    _deferFunct.bind( this )();
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/82087213261641c8e9f2487be77af9e7671f8bf6